.appveyor.yml

```diff
@@ -0,0 +1,103 @@
+build: false
+shallow_clone: true
+platform:
+  - x64
+matrix:
+  fast_finish: true
+clone_folder: C:\projects\dbal
+branches:
+  except:
+    - gh-pages
+
+## Build matrix for lowest and highest possible targets
+environment:
+  dependencies: current
+  matrix:
+  - db: sqlite
+    provider: sqlite
+    php_ver_target: 7.0
+  - db: sqlite
+    provider: sqlite
+    php_ver_target: 7.1
+  - db: mysql
+    provider: mysql
+    php_ver_target: 7.0
+  - db: mysql
+    provider: mysql
+    php_ver_target: 7.1
+  - db: postgresql94
+    provider: postgresql94
+    php_ver_target: 7.0
+  - db: postgresql94
+    provider: postgresql94
+    php_ver_target: 7.1
+  - db: mssql
+    provider: mssql
+    php_ver_target: 7.0
+  - db: mssql
+    provider: mssql
+    php_ver_target: 7.1
+
+## Cache composer bits
+cache:
+    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
+init:
+  - SET PATH=C:\Program Files\OpenSSL;c:\tools\php;%PATH%
+  - SET COMPOSER_NO_INTERACTION=1
+  - SET PHP=1
+  - SET ANSICON=121x90 (121x90)
+services:
+  - mssql2014
+  - mysql
+  - postgresql94
+  - iis
+
+## Install PHP and composer, and run the appropriate composer command
+install:
+    - IF EXIST c:\tools\php (SET PHP=0)
```

Comment:
I'm following a guide on the setup of php on appveyor. I've also been wondering If it's necessary or superfluous. https://blog.wyrihaximus.net/2016/11/running-php-unit-tests-on-windows-using-appveyor-and-chocolatey/